bigquery_etl/query_scheduling/templates/airflow_dag.j2

```diff
@@ -149,11 +149,12 @@ with DAG('{{ name }}', default_args=default_args{%+ if schedule_interval != None
         table_id={{ table_name | format_repr }},
         partition_id={{ table_partition_sensor_task.partition_id | format_repr }},
         gcp_conn_id='google_cloud_shared_prod',
-        deferrable=True,
+        deferrable=False,
+        mode='reschedule',
         {% if table_partition_sensor_task.poke_interval != None -%}
         poke_interval={{ table_partition_sensor_task.poke_interval | format_timedelta | format_repr }},
         {% else -%}
-        poke_interval=datetime.timedelta(minutes=5),
+        poke_interval=datetime.timedelta(minutes=15),
```

Comment:
I increased the default poke interval because starting up a Kubernetes pod every time the sensor runs is more resource intensive than when it was in deferrable mode (I think it takes around a minute to start each pod).